openassessment/xblock/static/js/openassessment-lms.min.js

```diff
@@ -1,3 +1,3 @@
-}).fail(function(){view.baseView.showLoadError("student-training")})},installHandlers:function(){var sel=$(".step--student-training",this.element);var view=this;this.baseView.setUpCollapseExpand(sel);var rubricSelector=$(".student-training--001__assessment",this.element);if(rubricSelector.size()>0){var rubricElement=rubricSelector.get(0);this.rubric=new OpenAssessment.Rubric(rubricElement)}if(this.rubric!==null){this.rubric.canSubmitCallback($.proxy(this.assessButtonEnabled,this))}sel.find(".student-training--001__assessment__submit").click(function(eventObject){eventObject.preventDefault();view.assess()})},assess:function(){this.assessButtonEnabled(false);var options={};if(this.rubric!==null){options=this.rubric.optionsSelected()}var view=this;var baseView=this.baseView;var usageID=baseView.getUsageID();this.server.trainingAssess(options).done(function(corrections){var incorrect=$(".openassessment__student-training--incorrect",view.element);var instructions=$(".openassessment__student-training--instructions",view.element);if(!view.rubric.showCorrections(corrections)){view.load(usageID);baseView.loadAssessmentModules(usageID);incorrect.addClass("is--hidden");instructions.removeClass("is--hidden")}else{instructions.addClass("is--hidden");incorrect.removeClass("is--hidden")}baseView.scrollToTop(".step--student-training")}).fail(function(errMsg){baseView.toggleActionError("student-training",errMsg);view.assessButtonEnabled(true)})},assessButtonEnabled:function(isEnabled){return this.baseView.buttonEnabled(".student-tra
```

Comment:
why is this min file getting pushed? doesn't our deployment generate new ones?

Reply:
as per my understanding as going through the code my findings are that I would not compile static files of external packages.